en/docs/develop/creating-artifacts/creating-an-api.md

```diff
@@ -233,45 +234,64 @@ The service's metadata is used by the API management runtime to generate the API
 
 -->
 
-### Design the integration
+## Design the integration
+
+1. Open the **Resource View** of the API resource.
+
+    1. Go to **MI Project Explorer** > **APIs**.
+    2. Under the API you created, click the API resource to open the **Resource View** of the API resource.
+
+2. Click on the **+** icon below the API resource to open the mediator palette.
+
+3. Add the required mediators and connectors to the API resource and design the integration flow.
+
+## Update properties
+
+### Update the API properties
```

Comment:
Let's rename this to configuration

Reply:
Fixed via https://github.com/wso2/docs-mi/pull/1151/commits/4d790beff54453c1054d81f07b4a360bbeada5b4.